### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by develogix**
Progress towards #9

### CODE REVIEW

File: commands/beer-lookup.js [hunk 1]
```diff
@@ -10,8 +10,22 @@ exports.brew = {
...
+			msg.channel.send({
+				embed: {
+					color: GReYBot.Config.defaultEmbedColor,
+					author: {
+						name: 'BreweryDB',
+						url: 'http://www.brewerydb.com/',
+						icon_url: 'https://emojipedia-us.s3.amazonaws.com/thumbs/120/twitter/103/beer-mug_1f37a.png'
+					},
+					description: 'How about asking for something specific?',
+					fields: fields
```
Comment (naterchrdsn) [inline commands/beer-lookup.js]: the fields don't exist at this point. the variable isnt even initialized

File: commands/beer-lookup.js [hunk 2]
```diff
@@ -23,66 +37,70 @@ exports.brew = {
...
+							thumbnail: {
+								url: thumbnail
+							},
+							description: `\n${result.description}\n\n`,
+							fields: fields
+						}
 					});
 				} else {
 					msg.channel.send({
 						embed: {
```
Comment (naterchrdsn) [inline commands/beer-lookup.js:97]: if we're going to style the embed for a mistake in typing, we should style the response for no description

File: commands/cocktail-lookup.js [hunk 3]
```diff
@@ -78,15 +92,15 @@ exports.cocktail = {
...
-							description: `${response.data.drinks[0].strDrink} is a good drink, but I don\'t have a good way to describe it.`
+							description: `${response.data.drinks[0].strDrink} is a good drink, but I don't have a good way to describe it.`
 						}
 					});
 				}
 			} else {
 				msg.channel.send({
 					embed: {
 						color: GReYBot.Config.defaultEmbedColor,
-						description: `Damn, I've never heard of that.  Where do I need to go to find it?`
```
Comment (naterchrdsn) [inline commands/cocktail-lookup.js:89]: i still dont get how there were so many spacing issues... even after re-checking out the branch

File: commands/moderation.js [hunk 4]
```diff
@@ -1,165 +1,273 @@
...
-								}
-							}).then(message => message.delete(5000));
-						});
+exports.ban = {
+	usage: '<user> [days of messages to delete] [reason]',
+	description: 'bans the user, optionally deleting messages from them in the last x days',
+	process: function (msg, suffix) {
+		var args = suffix.split(' ');
+		if (args.length > 0 && args[0]) {
+			if (!msg.guild.me.hasPermission('BAN_MEMBERS', true) || !msg.guild.me.hasPermission('ADMINISTRATOR')) {
```
Comment (naterchrdsn) [inline commands/moderation.js]: this is not necessary. looking at the api you can see that .hasPermission accepts an "admin override" boolean. so the double-check is redundant.

File: commands/moderation.js [hunk 5]
```diff
@@ -1,165 +1,273 @@
...
+				}).then(message => message.delete(5000));
 
-					lastPruned = new Date().getTime();
-				}
+				return;
 			}
-			else {
-				var wait = Math.floor(GReYBot.Config.pruneInterval - (timeSinceLastPrune / 1000));
+
+			if (!msg.member.hasPermission('BAN_MEMBERS', true) || !msg.member.hasPermission('ADMINISTRATOR')) {
```
Comment (naterchrdsn) [inline commands/moderation.js]: same as above

File: commands/moderation.js [hunk 6]
```diff
@@ -1,165 +1,273 @@
...
+					color: GReYBot.Config.defaultEmbedColor,
+					description: 'You must specify a number of messages to prune.'
 				}
-			});
+			}).then(message => message.delete(5000));
+
+			return;
+		}
+
+		if (!msg.guild.me.hasPermission('KICK_MEMBERS', true) || !msg.guild.me.hasPermission('ADMINISTRATOR')) {
```
Comment (naterchrdsn) [inline commands/moderation.js]: same as above

File: commands/moderation.js [hunk 7]
```diff
@@ -1,165 +1,273 @@
...
+				embed: {
+					color: GReYBot.Config.defaultEmbedColor,
+					description: `I can't prune messages, ${msg.member}...`
+				}
+			}).then(message => message.delete(5000));
+
+			return;
+		}
+
+		if (!msg.member.hasPermission('MANAGE_MESSAGES', true) || !msg.member.hasPermission('ADMINISTRATOR')) {
```
Comment (naterchrdsn) [inline commands/moderation.js]: same as above

**Review by naterchrdsn — CHANGES_REQUESTED** [review]

**Comment by naterchrdsn** [comment]
https://discord.js.org/#/docs/main/stable/class/GuildMember?scrollTo=hasPermission

**Comment by develogix** [comment]
Here we have a case study of how _not_ to use Git... don't code on one computer and test on another.

**Review by naterchrdsn — APPROVED** [review]

**Pull request merged**